plugins/woocommerce/src/Admin/DataSourcePoller.php

```diff
@@ -133,7 +133,7 @@ public function read_specs_from_data_sources() {
 			$this->merge_specs( $specs_from_data_source, $specs, $url );
 		}
 
-		$specs_group            = get_transient( $this->args['transient_name'] ) ?? array();
+		$specs_group            = get_transient( $this->args['transient_name'] ) ?: array();
```

Comment:
Hi @markjaquith, thanks for your contribution. Unfortunately [the short ternary operator is forbidden in the WordPress coding standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#ternary-operator) 🤷🏻‍♂️ So unfortunately a more verbose alternative is required:

```suggestion
		$specs_group = get_transient( $this->args['transient_name'] );
		if ( false === $specs_group ) {
			$specs_group = array();
		}
```

Also could you please add testing instructions to the pull request description? Thank you!